tests/debug/mod.rs

```diff
@@ -36,7 +36,7 @@ fn debug_fmt(f: MakeFunction) -> Value {
     let type_name = receiver.get_type_name();
 
     match receiver.data() {
-        Data::Struct(receiver) => match receiver {
+        Data::Struct(receiver, ..) => match receiver {
```

Comment:
I would prefer not to make this change. Could you expose them through an accessor method on the appropriate type instead, with the content stored in thread local?

Reply:
Actually the content can go inside UnitStruct/TupleStruct/StructStruct, with an accessor `fn attrs(&self) -> &[Attribute]` on Data, Struct, UnitStruct, TupleStruct, StructStruct as already exists on reflect::Field.